bosta/base.py

```diff
@@ -36,7 +36,7 @@ def airwaybill_get(self,
         arg = airwaybill.GetAirWaybillArg(_id)
         r = self.request(
             airwaybill.get,
-            'airwaybill',
+            'deliveries/awb',
```

Comment:
Was this change auto-generated?

Reply:
Doesn't look like it.  I guess it's either we change the namespace to produce this desired result, or we rewrite the client to use a different attribute to build the endpoint URL.